meta-lxatac-bsp/recipes-kernel/linux/files/defconfig

```diff
@@ -5891,6 +5892,7 @@ CONFIG_FUSE_FS=y
 # CONFIG_CUSE is not set
 # CONFIG_VIRTIO_FS is not set
 CONFIG_FUSE_PASSTHROUGH=y
+CONFIG_FUSE_IO_URING=y
```

Comment:
DO you use this on the LXATAC?

Reply:
Maybe?
The TAC is designed to be used interactively and a user may decide to use a FUSE filesystem that uses `io_uring`.